src/bootstrap/package-json.js

```diff
@@ -3,11 +3,11 @@ import json from '../util/json'
 
 const saguiScripts = {
   'start': 'npm run develop',
-  'test': 'NODE_ENV=test sagui test',
-  'test-watch': 'NODE_ENV=test sagui test --watch',
-  'develop': 'sagui develop',
-  'build': 'sagui build',
-  'dist': 'NODE_ENV=production sagui dist'
+  'test': 'NODE_ENV=test karma start',
+  'test:watch': 'npm test -- --no-single-run --auto-watch',
+  'develop': 'webpack-dev-server --port 3000 --host 0.0.0.0',
+  'build': 'webpack',
+  'dist': 'NODE_ENV=production webpack -p'
```

Comment:
Shouldn't `test`, `develop`, `build` and `dist` also be removed from the codebase if they're not to be used anymore?